.gitignore

```diff
@@ -10,3 +10,7 @@
 
 # rspec failure tracking
 .rspec_status
+
+# editors
+/.idea
+/.vscode
```

Comment:
Please configure EOL configuration to break line automatically in your editor :)